src/CartLatticeAccess.hpp.Rt

```diff
@@ -253,18 +250,22 @@ resolve.symmetries = function(D) {
 
 ?>
 
+CudaDeviceFunction flag_t getContainerType(const CartLatticeContainer& container, int x, int y, int z) {
+  return container.NodeType[(x + container.nx*(y + container.ny*z))];
+}
+
 template < class x_t, class y_t, class z_t >
-class LatticeAccess {
+class CartLatticeAccess {
   private:
   const x_t x;
   const y_t y;
   const z_t z;
   const flag_t nt;
   const range_int<0,1,0,1> nx, ny, nz;
   public:
-  CudaDeviceFunction LatticeAccess(const int& x_, const int& y_, const int& z_) :
+  CudaDeviceFunction CartLatticeAccess(const int& x_, const int& y_, const int& z_) :
     x(x_),y(y_),z(z_),
-    nt(constContainer.getType(x,y,z)),
+    nt{getContainerType(constContainer, x, y, z)},
```

Comment:
Can we use normal brackets in object initializations? There is not a lot of difference and let's just make it consistent with the rest of the code.